src/task/style.css

```diff
@@ -242,6 +242,11 @@ footer {
   margin: 0;
 }
 
+#list-items {
+  max-height: 200px;
+  overflow-y: scroll;
```

Comment:
What about overflow-x: If I change css property in existing codebase I am seeing scroll in x axis. Can you check

Reply:
Resolved.